pyscf/adc/uadc_ao2mo.py

```diff
@@ -231,6 +219,8 @@ def transform_integrals_outcore(myadc):
         if chnk_size <= 0 :
             chnk_size = 1
 
+        chnk_size = 10
```

Comment:
Is this remnants of a debugging process??? Shouldn't be here. Check everywhere else.

Reply:
Removed